app/models/curation_concerns/state_workflow.rb

```diff
@@ -7,7 +7,7 @@ def initialize(state)
     attr_reader :state
 
     def pending?
-      state == :pending
+      state == :pending || state == ::RDF::URI("http://fedora.info/definitions/1/0/access/ObjState#inactive")
```

Comment:
I don't think we need this earlier access at all. In fact, I'm not sure we need this class at all.  Can we just move this check into the indexer?  Also use `Vocab::FedoraResourceStatus.inactive` instead of creating a new URI.